bundles/org.openhab.binding.haassohnpelletstove/README.md

```diff
@@ -0,0 +1,67 @@
+# Haas Sohn Pellet Stove Binding
+
+The binding for Haassohnpelletstove communicates with a Haas and Sohn Pelletstove through the optional
+WIFI module. More information about the WIFI module can be found here: https://www.haassohn.com/de/ihr-plus/WLAN-Funktion
+
+## Supported Things
+
+| Things | Description  | Thing Type |
+|--------|--------------|------------|
+| haassohnpelletstove | Control of a Haas & Sohn Pellet Stove| oven|
+
+
+## Thing Configuration
+
+In general two parameters are required. The IP-Address of the WIFI-Modul of the Stove in the local Network and the Access PIN of the Stove.
+The PIN can be found directly at the stove under the Menue/Network/WLAN-PIN
+
+```
+Thing haassohnpelletstove:oven:myOven "Pelletstove"  [ hostIP="192.168.0.23", hostPIN="1234"]
+```
+
+## Channels
+
+The following channels are yet supported:
+
+
+| Channel | Type  | Access| Description|
+|---------|-------|-------|------------|
+| power| Switch | read/write|Turn the stove on/off|
+|channelIsTemp|Number:Temperature|read|Receives the actual temperature of the stove|
+|channelSpTemp|Number:Temperature|read/write|Receives and sets the target temperature of the stove|
```

Comment:
Can you fix the formatting of the table? There are several online markdown table formatters on the net.

Reply:
Fixed it before. Looks perfect already in the repo. Have a look here [https://github.com/chingon007/openhab-addons/tree/main/bundles/org.openhab.binding.haassohnpelletstove](url)

Reply:
The link you posted is broken. I see it still unformatted in your branch: https://raw.githubusercontent.com/chingon007/openhab-addons/main/bundles/org.openhab.binding.haassohnpelletstove/README.md

Reply:
This is indeed correct. If you open it in a browser, it looks perfect (see screenshot). This is how I got it out of a online formatter. 
![2021-02-21 10_05_24-openhab-addons_README md at main · chingon007_openhab-addons](https://user-images.githubusercontent.com/76529461/108620448-96a96900-742c-11eb-99ad-44651e1c28f6.png)